src/main/java/org/jabref/logic/importer/fetcher/IEEE.java

```diff
@@ -224,12 +232,28 @@ public String getName() {
         return Optional.of(HelpFile.FETCHER_IEEEXPLORE);
     }
 
+    /**
+     * Gets IEEE api key, use key if it customized it in the preferences, otherwise use the default
+     *
+     * @return IEEE API Key
+     */
+    private String getApiKey() {
+        String apiKey = API_KEY;
+        if (preferencesService != null) {
```

Comment:
You can omit the null check here, that is already covered with Objects.requireNonNull

Reply:
fixed.